docs/src/Box.doc.js

```diff
@@ -494,6 +502,48 @@ card(
   />
 );
 
+card(
+  <Card
+    description={`
+    Borders are controlled by the "borderSize" prop. Setting a size enables a solid light gray color in that width.
+  `}
+    id="border"
+    name="Borders"
+  >
+    <Box display="flex" wrap>
```

Comment:
@jennyscript should we use the `Combinations` component like we do in other places?